proj/src/middleware/projectsMiddleware.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const projectsMiddleware = ({dispatch}) => next => action => {
+  if (!action.meta || action.meta.type !== 'api') {
+    console.log('going to next')
+    return next(action);
+  }
+  
+  axios.get(action.meta.url)
+    .then(({data}) => dispatch(action.payload.onSuccess(data)))
```

Comment:
make the success and error handlers able to receive arrays of actions (onSuccess and onError can one or many actions)